exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -1,7 +1,157 @@
-## READ Operations
+# CRUD Operations
+
+This document deals with basic create, read, update and delete operations using the mongo shell. 
+
+[Read more](https://docs.mongodb.com/manual/crud/) :book: about the CRUD operations here. 
+
+## Create Operations
+
+A MongoDB database is made up of 1 or more collections which are in turn made up of 1 or more documents. A document has a `key: value` structure, similar to JSON objects. In the following exercises, we'll learn how to insert documents using the following operations:
+* `insertOne()`
+* `insertMany()` 
+
+**Exercise 1** :computer: 
+
+* Verify what database you are currently using (should be `test`, this is the default.)
+
+    ```javascript
+    db
+    ```
+
+* Let's create a new database called `employee`. 
+    ```javascript
+    use employee_db
+    ```
+
+:arrow_right: Note that `use` creates a new database, if one does not already exist. 
+
+* Next, let's create a collection called `employee_info`.
+
+    ```javascript
+    db.createCollection("employee_info")
+    ```
+    
+* Let's confirm that the collection was successfully created.
+
+    ```javascript
+    show collections
+    ```
+:arrow_right: Similarly, you can also use `show dbs` to obtain a list of the all the databases. 
+
+* Let's try to insert the following `key:value` pairs as a document in our collection. 
+
+    {
```

Comment:
What about a table showing this information before the object?  This would allow them to see table data converted to JSON.  

Also a block of JSON is confusing here... what is this object, why does it exist.  We should preface whey they are seeing JSON right now.  Maybe something like "below you will find the data we are going to insert.  This is what it looks like as a table, but we need it as an object... oh hey, here's the object, okay let's insert it 👍 "